src/libs/icon-dict.js

```diff
@@ -49,6 +50,7 @@ const iconDict = {
   copy: fa(faClone),
   'copy-to-clipboard': fa(faClipboard),
   download: fa(faDownload),
+  downloadLight: custom(downloadLight),
```

Comment:
Not sure why some are camelCase and some are kebab-case.

Reply:
Legacy stuff, just wasn't ever cleaned up.